CodeStarterCamp_Week1/main.swift

```diff
@@ -6,7 +6,16 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
-
-print("Hello, World!")
+func drawIceCream(creamLength: Int) {
+    for _ in 1...creamLength{
+        print("***********")
+    }
+} // 아이스크림을 * 11개로 8줄 출력
+func drawIceCreamBar(barLength: Int) {
+    for _ in 1...barLength{
+        print("    | |    ")
+    }
```

Comment:
```suggestion
    for _ in 1...barLength {
        print("    | |    ")
    }
```
틀린건 절대 아니지만 보통 띄어서 쓰는 것 같습니다 😎